src/actions/__tests__/read-rule.test.js

```diff
@@ -74,4 +74,45 @@ describe('readRule()', () => {
       runReadRule(undefined, docs.replace('\n', '\r\n')).newDocs
     ).toMatchSnapshot()
   })
+
+  it('mixed windows and unix newlines', () => {
+    const docs = `# ${description} (${name})\nSome random test:\r\n\r\n`
```

Comment:
Would it make sense to extract the `# ${description} (${name})` out to a `heading` variable so it doesn’t have to be repeated in each test?